did/base.py

```diff
@@ -150,6 +150,22 @@ def width(self):
         except (NoOptionError, NoSectionError):
             return MAX_WIDTH
 
+    @property
+    def border_char(self):
+        """ Border character to use for the report """
```

Comment:
I'm not sure about the naming as `border` confused me a bit when reading the pull request. What about `separator`? Does that sound ok? @lukaszachy, any better idea?

Reply:
Happy to change the name to whatever you prefer

Reply:
I won't update the docs until everything is finalised

Reply:
Ok, let's use `separator` instead of `border_char` and `separator_width` for `border_count`. Thanks.

Reply:
Done

Reply:
Doco updated

Reply:
I was concerned about renaming `border_char` to `separator` as it might cause confusion with other uses, for example: `emails = utils.split(emails, separator=re.compile(r"\s*,\s*"))` .. wdyt?

Reply:
Done